pgctl/cli.py

```diff
@@ -347,7 +346,7 @@ def services(self):
             for alias in self.pgconf['services']
             for service_name in self._expand_aliases(alias)
         ]
-        return uniq(services)
+        return tuple(sorted(set(services)))
```

Comment:
a) since Service is a nametuple, it inherits tuple sorting. This would result in a sort by path essentially.

I'd much rather have the input order.

Put this back how it was, please. The rest is fine.